### PR TITLE
GanttProject.GanttProject version 3.1.3100

### PR DESCRIPTION
**Pull request by techflashYT**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/15945)

### CODE REVIEW

**Comment by sanikdah** [comment]
(My comment from #15940 ) Same here.

**Comment by denelon** [comment]
@sanikdah one of our moderators needs to review to apply the "Moderator-Approved" label.

**Comment by denelon** [comment]
From the site https://www.ganttproject.biz/ A better ShortDescription would be "Free desktop project scheduling app".

When you install with this manifest what is the output from `winget list gantt`?

**Comment by sanikdah** [comment]
> From the site https://www.ganttproject.biz/ A better ShortDescription would be "Free desktop project scheduling app".
> 
> When you install with this manifest what is the output from `winget list gantt`?

Name         Id           Version
---------------------------------
GanttProject GanttProject Unknown

**Comment by denelon** [comment]
@sanikdah, OK based on that this is one of those packages that will likely always show an upgrade is available even when the user has the latest version on their system.

File: manifests/g/GanttProject/GanttProject/3.1.3100/GanttProject.GanttProject.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: GanttProject.GanttProject
+PackageVersion: 3.1.3100
+PackageLocale: en-US
+Publisher: GanttProject
+PackageName: GanttProject
+License: GNU GPL
+ShortDescription: A project manager.
```
Comment (denelon) [inline manifests/g/GanttProject/GanttProject/3.1.3100/GanttProject.GanttProject.locale.en-US.yaml:9]:
```suggestion
ShortDescription: Free desktop project scheduling app

```

**Review by denelon — CHANGES_REQUESTED** [review]

**Comment by sanikdah** [comment]
And there you go in here too.

**Comment by sanikdah** [comment]
> @sanikdah, OK based on that this is one of those packages that will likely always show an upgrade is available even when the user has the latest version on their system.

Yeah, nothing anyone can do about that, its up to the devs of the programs to fix that I'm pretty sure, maybe we should add a warning when running "winget upgrade --all" as long as there are programs that have a version of unknown?

**Review by denelon — APPROVED** [review]

**Pull request merged**